sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -153,14 +160,12 @@ def update(
     self._inference_request_batch_byte_size.update(examples_byte_size)
 
 
-class _RunInferenceDoFn(beam.DoFn):
+class _RunInferenceDoFn(beam.DoFn, Generic[ExampleT, PredictionT]):
   """A DoFn implementation generic to frameworks."""
-  def __init__(self, model_loader: ModelLoader, clock=None):
+  def __init__(
+      self, model_loader: ModelLoader[ExampleT, PredictionT, Any], clock=None):
```

Comment:
Why are we using `Any` instead of `ModelT` here? (same with line 99)

Reply:
This class imposes no constraints in the model type and doesn't use it in any way.